### PR TITLE
Add support of OC slot-id path

### PR DESCRIPTION
**Pull request by arulkumarsekar**

### CODE REVIEW

**Review by dplore — DISMISSED** [review]
LGTM for test intent.

File: feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go [hunk 1]
```diff
@@ -528,7 +530,12 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []*oc.Co
 					t.Errorf("Component %s Description: got empty string, want non-empty string", cName)
 				}
 			}
-
+			if p.slotIDValidation && card.GetType() == oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_LINECARD {
```
Comment (greg-dennis) [inline feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go:533]: Get rid of slowIDValidation and just do the validation whenever the card type is oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_LINECARD? It looks like that wouldn't change the behavior for any test.

If you really think the validation ought not to be automatic, then I would get rid of the "card.GetType() == oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_LINECARD" condition. That would at least prevent someone from adding extraneous `slotIDValidation: true` fields.

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go (hunk 1 above)
Comment (arulkumarsekar) [reply feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go]: @greg-dennis updated. can you check now pls?

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**